<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cim">
        <property name="text">
         <string>Choose image</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="tedit"/>
    </item>
    <item>
     <widget class="QPushButton" name="sim2">
      <property name="text">
       <string>Show image 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="sim3">
      <property name="text">
       <string>Show image 3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="sim4">
      <property name="text">
       <string>Show image 4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="sim5">
      <property name="text">
       <string>Show image 5</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="sim6">
      <property name="text">
       <string>Show image 6</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
